sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -89,6 +89,8 @@ private[spark] object HiveUtils extends Logging {
       |   Use Hive jars configured by `spark.sql.hive.metastore.jars.path`
       |   in comma separated format. Support both local or remote paths.
```

Comment:
How about we just fix in 3. and 4., and adding one line for each item like:

```scala
      |   in comma separated format. Support both local or remote paths. The provided jars
      |   should be the same version as ${HIVE_METASTORE_VERSION}.
```

?

Reply:
> How about we just fix in 3. and 4., and adding one line for each item like:
> 
> ```scala
>       |   in comma separated format. Support both local or remote paths. The provided jars
>       |   should be the same version as ${HIVE_METASTORE_VERSION}.
> ```
> 
> ?

Done